src/main/scala/org/broadinstitute/dsde/agora/server/model/AgoraEntity.scala

```diff
@@ -25,7 +38,8 @@ case class AgoraEntity(@(ApiModelProperty@field)(required = false, value = "The
                        @(ApiModelProperty@field)(required = false, value = "The method payload")
                        payload: Option[String] = None,
                        @(ApiModelProperty@field)(required = false, value = "URI for method details")
-                       url: Option[String] = None
+                       url: Option[String] = None,
+                       entityType: Option[AgoraEntityType.EntityType]= None
```

Comment:
This is more for when an entity moves through persistence -> domain -> web service. We need each layer to know what type it is without having to infer it.

I suppose it doesn't need to be stored in the database directly as it can be inferred from the collection, but it would be more work to exclude it. 

For the methods post end point we would need to add it as an api model property otherwise we would need to pull it out of the WDL. I'm fine with either... opinions?